k8s/namespaces/fees-pay/fees-register-frontend/perftest.yaml

```diff
@@ -3,25 +3,11 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: fees-register-frontend
-  namespace: fees-pay
 spec:
   releaseName: fees-register-frontend
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/fees-register-frontend
   values:
     nodejs:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/fees-register/frontend:perftest #{"$imagepolicy": "flux-system:perftest-fees-register-frontend"}
       imagePullPolicy: Always
```

Comment:
is this needed?

Reply:
as this is common so i kept it in fees-register-frontend.yaml  and removed from env specific files 
       replicas: 2
      useInterpodAntiAffinity: true